OWNERS

```diff
@@ -4,18 +4,10 @@ approvers:
 - gyliu513
 - jichenjc
 - mkumatag
-
-reviewers:
```

Comment:
Why are we removing the reviewers section here?

/hold

Reply:
> Why are we removing the reviewers section here?
> 
> /hold

@dims Do we need to remove reviewers section?

Reply:
`emeritus_reviewers` is not a valid key, so that section should be removed. we should leave `reviewers` section as-is. `reviewers` is a valid section.

Reply:
@dims Thanks for your reply, I have resumed the reviewers section. @mkumatag Thanks.